### PR TITLE
Added Cloud SQL MySQL connectivity samples for mysql library. 

### PR DESCRIPTION
**Pull request by kurtisvg**
Adding some base mysql connectivity samples.

### CODE REVIEW

File: cloud-sql/mysql/mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+export GOOGLE_APPLICATION_CREDENTIALS=/path/to/service/account/key.json
+export CLOUD_SQL_CONNECTION_NAME='<MY-PROJECT>:<INSTANCE-REGION>:<MY-DATABASE>'
+export DB_USER='my-db-user'
+export DB_PASS='my-db-pass'
+export DB_NAME='my_db'
+```
+Note: Saving credentials in environment variables is convenient, but not secure - consider a more
+secure solution such as [Cloud KMS](https://cloud.google.com/kms/) to help keep secrets safe.
+
+## Running locally
```
Comment (fhinkel) [inline cloud-sql/mysql/mysql/README.md]: Seems to be in here twice

File: cloud-sql/mysql/mysql/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+To launch your browser and view the app at https://[YOUR_PROJECT_ID].appspot.com, run the following
+command:
+```bash
+gcloud app browse
+```
+
+## Deploy to Google App Engine Flexible
+
+First, update `app.standard.yaml` with the correct values to pass the environment
```
Comment (fhinkel) [inline cloud-sql/mysql/mysql/README.md]: shouldn't this be a different yml file?

File: cloud-sql/mysql/mysql/README.md (hunk 1 above)
Comment (kurtisvg) [reply cloud-sql/mysql/mysql/README.md]: Removed

**Comment by kurtisvg** [comment]
@fhinkel Ready for another pass.

**Pull request merged**